bin/git-forgit

```diff
@@ -1002,6 +1003,130 @@ _forgit_ignore_clean() {
+}
+
+_forgit_worktree_jump() {
+    _forgit_inside_work_tree || _forgit_inside_git_dir || return 1
+    local worktree_list count tree opts
+    worktree_list=$(git worktree list | grep -vE "prunable$" | awk '{print $1}' | _forgit_filter_existing_paths)
+
+    count=$(echo "$worktree_list" | wc -l)
+    [[ $count -eq 1 ]] && return 1
+
+    opts="
+        $FORGIT_FZF_DEFAULT_OPTS
+        +s +m --tiebreak=index
+        --preview=\"$FORGIT worktree_preview {1}\"
+        $FORGIT_WORKTREE_JUMP_FZF_OPTS
+    "
+
+    tree=$(echo "$worktree_list" | FZF_DEFAULT_OPTS="$opts" fzf)
+    [[ -z "$tree" ]] && return 1
+    echo "$tree"
+}
+
+_forgit_git_worktree_lock() {
+    _forgit_worktree_lock_git_opts=()
+    _forgit_parse_array _forgit_worktree_lock_git_opts "$FORGIT_WORKTREE_LOCK_GIT_OPTS"
+    git worktree lock "${_forgit_worktree_lock_git_opts[@]}" "$@"
+}
+
+_forgit_worktree_lock() {
+    _forgit_inside_work_tree || _forgit_inside_git_dir || return 1
+    if [[ $# -ne 0 ]]; then
+        git worktree lock "$@"
+        worktree_lock_status=$?
+        return $worktree_lock_status
+    fi
+    local tree opts
+
+    opts="
+        $FORGIT_FZF_DEFAULT_OPTS
+        +s +m --tiebreak=index
+        --preview=\"$FORGIT worktree_preview {1}\"
+        $FORGIT_WORKTREE_LOCK_FZF_OPTS
+    "
+
+    tree=$(git worktree list | grep -vE "\(bare\)|locked" | awk '{print $1}' | FZF_DEFAULT_OPTS="$opts" fzf)
+    [[ -z "$tree" ]] && return 1
+    _forgit_git_worktree_lock "$tree"
+}
+
+_forgit_git_worktree_remove() {
+    _forgit_worktree_remove_git_opts=()
+    _forgit_parse_array _forgit_worktree_remove_git_opts "$FORGIT_WORKTREE_REMOVE_GIT_OPTS"
+    git worktree remove "${_forgit_worktree_remove_git_opts[@]}" "$@"
+}
+
+_forgit_worktree_remove() {
+    _forgit_inside_work_tree || _forgit_inside_git_dir || return 1
+    if [[ $# -ne 0 ]]; then
+        git worktree remove "$@"
+        worktree_remove_status=$?
+        return $worktree_remove_status
+    fi
+    local worktree_list tree opts
+    worktree_list=$(git worktree list | grep -v "(bare)")
+
+    count=$(echo "$worktree_list" | wc -l)
+    [[ $count -eq 1 ]] && return 1
+
+    opts="
+        $FORGIT_FZF_DEFAULT_OPTS
+        +s +m --tiebreak=index
+        --preview=\"$FORGIT worktree_preview {1}\"
+        $FORGIT_WORKTREE_REMOVE_FZF_OPTS
+    "
+
+    tree=$(echo "$worktree_list" | awk '{print $1}' | FZF_DEFAULT_OPTS="$opts" fzf)
+    [[ -z "$tree" ]] && return 1
+    _forgit_git_worktree_remove "$tree"
+}
+
+_forgit_worktree_unlock() {
+    _forgit_inside_work_tree || _forgit_inside_git_dir || return 1
+    if [[ $# -ne 0 ]]; then
+        git worktree unlock "$@"
+        worktree_unlock_status=$?
+        return $worktree_unlock_status
+    fi
+    local worktree_list tree opts
+
+    worktree_list=$(git worktree list | grep -v "(bare)" | grep -E "locked$")
+    count=$(echo "$worktree_list" | wc -l)
+    [[ $count -eq 1 ]] && return 1
```

Comment:
```suggestion
    [[ $count -eq 0 ]] && return 1
```
`$count` being 1 is actually fine here, because we filter to only include locked workspaces. Without this change it's impossible to unlock a workspace when there is only one locked worspace.